README.md

```diff
@@ -74,38 +74,86 @@ docker push codeforafrica/ckan:latest
 
 ---
 
-## Deployment (Untested)
+## Deployment
 
 We use [dokku](http://dokku.viewdocs.io/dokku/) for deployment so you'd need to install and set it up first.
 
+**Install Dokku on MAC:**
+
+Dokku runs on Linux so MAC users will have to setup dokku on a virtual machine
+
+```brew cask install virtualbox
+brew cask install vagrant
+git clone https://github.com/dokku/dokku.git
+cd dokku
+```
+
+Assign IP address 10.0.0.2 to dokku.me
```

Comment:
I feel it will be much clearer to someone if this step was more explicitly outlined, together with maybe a screenshot. 
After running `sudo nano /private/etc/hosts`, the user should add a line to the end of the file with the IP address.
`10.0.0.2        dokku.me`

![Uploading Screen Shot 2017-09-19 at 12.09.13.png…]()

Reply:
Great suggestion. Thanks, I will add this

Reply:
Great.